src/report.rs

```diff
@@ -44,7 +46,50 @@ impl Report {
       format!("unix:{:?}", platform.os_type).into()
     };
 
-    let backtrace = format!("{:#?}", Backtrace::new());
+    //We skip non-user code frames, including Backtrace::new()
+    const SKIP_FRAMES_NUM: usize = 8;
+    //Code is based on backtrace source
+    const HEX_WIDTH: usize = mem::size_of::<usize>() * 2 + 2;
+
+    let mut backtrace = String::new();
+
+    for (idx, frame) in Backtrace::new()
+      .frames()
+      .iter()
+      .skip(SKIP_FRAMES_NUM)
+      .enumerate()
+    {
+      let ip = frame.ip();
+      let _ = write!(backtrace, "\n{:4}: {:2$?}", idx, ip, HEX_WIDTH);
+
+      let symbols = frame.symbols();
+      if symbols.len() == 0 {
```

Comment:
Just gonna put the clippy warning in here:

You should use  `symbols.is_empty()` because it's more concise than manually checking length